src/test/regress/expected/relation_access_tracking_single_node.out

```diff
@@ -0,0 +1,1090 @@
+---------------------------------------------------------------------
```

Comment:
this produces the same output with src/test/regress/expected/relation_access_tracking.out

which is what we want

Reply:
Then should we instead add this into arbitrary config tests ?